Source/JavaScriptCore/interpreter/Interpreter.h

```diff
@@ -157,7 +159,7 @@ using JSOrWasmInstruction = std::variant<const JSInstruction*, const WasmInstruc
 
         NEVER_INLINE CatchInfo unwind(VM&, CallFrame*&, Exception*);
         void notifyDebuggerOfExceptionToBeThrown(VM&, JSGlobalObject*, CallFrame*, Exception*);
-        NEVER_INLINE void debug(CallFrame*, DebugHookType);
+        NEVER_INLINE void debug(CallFrame*, DebugHookType, JSCell* data);
```

Comment:
Let's just take JSValue for now as we are passing jsUndefined when it is not a generator.